packages/ast-spec/src/declaration/TSImportEqualsDeclaration/fixtures/entity-name-many/snapshots/5-AST-Alignment-AST.shot

```diff
@@ -2,5 +2,88 @@
 
 exports[`AST Fixtures declaration TSImportEqualsDeclaration entity-name-many AST Alignment - AST 1`] = `
 "Snapshot Diff:
-Compared values have no visual difference."
+- TSESTree
++ Babel
+
+  Program {
+    type: 'Program',
+    body: Array [
+      TSImportEqualsDeclaration {
+        type: 'TSImportEqualsDeclaration',
+        id: Identifier {
+          type: 'Identifier',
+          name: 'F',
+
+          range: [7, 8],
+          loc: {
+            start: { column: 7, line: 1 },
+            end: { column: 8, line: 1 },
+          },
+        },
+        importKind: 'value',
++       isExport: false,
```

Comment:
@bradzacher turns out I wasn't done with this - there are still a few `.shot` files with `isExport: false`. `yarn jest spec.ts` in `packages/typescript-estree` shows the failures.

But, on astexplorer.net, `@babel/parser` and other Babel parsers throw an error trying to parse code like `import F = A.B.C;`

Am I missing something about how these are generated?

Reply:
I'll try the changes in https://github.com/typescript-eslint/typescript-eslint/issues/6065 instead.